contents/handbook/people/team-structure/pipeline/objectives.mdx

```diff
@@ -0,0 +1,69 @@
+- Objective: Develop Posthog CDP into a solution capable of replacing other CDPs for data-warehousing categories. Increase % of customers using “data out” apps from X to Y
```

Comment:
Nit: Not a fan of acronyms. I don't think CDP will be a term relevant for people joining or existing people unless previously explained and this document doesn't do so. This document should be self-serving and not require additional context to digest.